app/src/main/res/values/strings.xml

```diff
@@ -245,41 +245,41 @@
     <string name="biometric_prompt_store_credential_title">Save biometric recognition</string>
     <string name="biometric_prompt_store_credential_message">Warning: You still need to remember your master password if you use biometric recognition.</string>
-    <string name="biometric_prompt_extract_credential_title">Open database with biometric recognition</string>
-    <string name="biometric_prompt_extract_credential_message">Extract database credential with biometric data</string>
+    <string name="biometric_prompt_extract_credential_title">Database unlocking with biometry</string>
+    <string name="biometric_prompt_extract_credential_message">Fetch database credential using biometric data recognition</string>
     <string name="encrypted_value_stored">Encrypted password stored</string>
-    <string name="biometric_invalid_key">Can not read the biometric key. Please delete it and repeat the biometric recognition procedure.</string>
+    <string name="biometric_invalid_key">Retry biometric recognition</string>
     <string name="biometric_not_recognized">Could not recognize biometric</string>
     <!--This problem could be with scanning, or something else.-->
     <string name="biometric_scanning_error">Biometric error: %1$s</string>
     <string name="no_credentials_stored">This database does not have stored credential yet.</string>
-    <string name="credential_before_click_biometric_button">Type the password before clicking the biometric button.</string>
+    <string name="credential_before_click_biometric_button">Type in the password first.</string>
     <string name="database_history">History</string>
     <string name="menu_appearance_settings">Appearance</string>
     <string name="biometric">Biometric</string>
     <string name="general">General</string>
     <string name="autofill">Autofill</string>
     <string name="autofill_service_name">KeePassDX form autofilling</string>
     <string name="autofill_sign_in_prompt">Sign in with KeePassDX</string>
-    <string name="autofill_explanation_summary">Enable autofilling to quickly fill out forms in other apps</string>
+    <string name="autofill_explanation_summary">Fill out other app logins automatically</string>
     <string name="set_autofill_service_title">Set default autofill service</string>
     <string name="password_size_title">Generated password size</string>
     <string name="password_size_summary">Sets default size of the generated passwords</string>
     <string name="list_password_generator_options_title">Password characters</string>
     <string name="list_password_generator_options_summary">Set allowed password generator characters</string>
     <string name="database_opened">Database opened</string>
     <string name="clipboard">Clipboard</string>
-    <string name="clipboard_explanation_summary">Copy entry fields using the clipboard of your device</string>
+    <string name="clipboard_explanation_summary">Copy entry fields into the clipboard of your device</string>
```

Comment:
Not the same sense, copying to the clipboard is different from copying from the clipboard to a form.